sort.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+function bubbleSort($sort_array)
+{
+    if (empty($sort_array)) {
+        return [];
+    }
```

Comment:
What's this for?

Reply:
If the array should be empty then it will return immediately without doing anything else. We don't need to check anything if it's empty.

Reply:
Even without this, the function returns more-or-less immediately. Please explain why.

Reply:
Because after each loop we check if something was swapped, if not then we abort and return what is in the array no matter if they are already sorted or the array is completely empty.

Reply:
So please remove the if.